src/components/Alert.tsx

```diff
@@ -0,0 +1,20 @@
+'use client';
+
+import { useContext } from 'react';
+import { AlertContext } from '@/context/AlertContext';
+
+export const Alert = (): JSX.Element => {
+  const { showAlert } = useContext(AlertContext);
+
```

Comment:
lets add this instead:
```
if (!showAlert) return null;
return (
...
)

Reply:
updated in https://github.com/RR-Ardhika/murojaah/pull/7/commits/7e3a825fb939daf367835007e1506ec2381dffcb, thank you